components/chart/overview.md

```diff
@@ -154,7 +154,8 @@ To execute Chart methods, obtain reference to the component instance via `@ref`.
 
 | Method  | Description |
 |---------|-------------|
-| Refresh | Use the method to programmatically re-render the Chart.  |
+| `Refresh` | Use the method to programmatically re-render the Chart.  |
+| `ResetDrilldownLevel` | Use the method to implement a custom breadcrumb for Chart navigation, to reset the Chart upon clicking a button, and more. For more information about the DrillDown feature, refer to the [DrillDown Charts article]({%slug chart-drilldown%}). |
```

Comment:
When explained like this, this method seems to do a lot of different things, but it's not used in an example anywhere. Please include it in the samples on the Drilldown page.